src/main/webapp/view/discussion/discussion.js

```diff
@@ -300,11 +300,10 @@ class DiscussionComment extends HTMLElement {
 // Custom element names must contain a hyphen.
 customElements.define('discussion-comment', DiscussionComment);
 
-/** Seeks discussion to `currentTimeSeconds`. */
-export function seekDiscussion(currentTimeSeconds) {
-  const currentTimeMilliseconds = secondsToMilliseconds(currentTimeSeconds);
-  updateNewCommentTimestamp(currentTimeMilliseconds);
-  const nearbyComments = getNearbyDiscussionComments(currentTimeMilliseconds);
+/** Seeks discussion to `currentTimeMs`. */
+export function seekDiscussion(currentTimeMs) {
```

Comment:
nit: should this be renamed currentVideoTimeMs?

Reply:
At the moment, yes.  But in the future it won't always be seeking to the current video time.